packages/story-editor/src/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -114,6 +128,15 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
     })
   );
 
+  useEffect(() => {
+    getFailingPages(pages, pageSize, currentPage).then((failedPages) => {
+      // getFailingPages returns an array of pages, since we only care
+      // about currentPage, we can grab the single page result.
+      const result = failedPages[0]?.result;
```

Comment:
currentPage will always be index 0?

Reply:
Yea. If we send in the `currentPage`, it will only check that single page and thus the returned array will have a length of one.